### PR TITLE
updated code for windows and linux and mac

### PR DESCRIPTION
**Pull request by Karthik2008**
I changed the code, check if it is fine

### CODE REVIEW

**Comment by jaideep1163** [comment]
Looks Good But There is a Problem,

There is Currently No Terminal For Windows So you Might have to change it a Bit 

Thanks Jaideep

**Comment by Karthik2008** [comment]
Oh, cool.
but they can run this using vs code right.....

btw, they can use any ide they have

and we have to also change the readme

thanks Karthik

**Comment by jaideep1163** [comment]
Sure
But The Problem is 
```py
def terminal():
        """ 
        This Function opens start menu,
        waits for 1 second, then opens terminal.
        """
        # presses the start menu button
        pg.hotkey('winleft')
        
        # waits for 1 second
        time.sleep(1)

        # types terminal with pauses of 0.3 seconds
        pg.typewrite('terminal\n', 0.3)

        # waits for 3 seconds
        time.sleep(3)
```

**Comment by Karthik2008** [comment]
yes, got the problem..

will fix it RN

File: main.py [hunk 1]
```diff
@@ -1,96 +1,260 @@
...
-    # holds command
-    pg.keyDown('command')
-    # presses space
-    pg.hotkey('space')
-    # releases command
-    pg.keyUp('command')
-    
-    # waits for 1 second
-    time.sleep(1)
+"""
```
Comment (jaideep1163) [inline main.py:7]: Lol

**Pull request closed without merging**